java/arcs/android/storage/database/DatabaseGarbageCollectionPeriodicTask.kt

```diff
@@ -15,7 +15,9 @@ import androidx.work.Worker
 import androidx.work.WorkerParameters
 import arcs.core.storage.driver.DatabaseDriverProvider
 import arcs.core.util.TaggedLog
+import kotlinx.coroutines.launch
 import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.supervisorScope
```

Comment:
unnecessary imports

Reply:
Oops, thanks! I had started doing this for garbage collection as well, but I don't think it's needed. Someone should check my work here...